core/modules/dashboard/includes/block.overview_user.inc

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @file
+ *
+ * User accounts block which displays a summary of user statistics including:
+ * - number of total user accounts.
+ * - number of active user accounts.
+ * - number ofblocked user accounts.
+ * - number of users in each role.
+ */
+class DashboardOverviewUserBlock extends Block {
+
+  /**
+   *  Sets block subject on block view.
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('User accounts');
+  }
+
+  /**
+   * 'Admin title' callback for the content type.
+   */
+  function getAdminTitle() {
+    return t('Overview of user accounts');
+  }
+
+  /**
+   * 'Admin info' callback for the content type.
+   */
+  function getAdminPreview() {
+    return t('Displays a summary of user statistics including: number of total users, active users, blocked users, and users in each role.');
+  }
+
+  /**
+   * Run-time rendering of the body of the block.
+   */
+  function getContent() {
+    $user_data = dashboard_get_user_overview($this->settings);
```

Comment:
Another place where the function is only called once. Perhaps just put everything in here?

Reply:
fixed by @quicksketch